.github/workflows/main.yml

```diff
@@ -170,6 +170,17 @@ jobs:
         echo "::remove-matcher owner=unittest::"
       if: runner.os == 'macOS' || runner.os == 'Linux'
 
+    - name: Test (Windows)
+      # Tests should complete in well under an hour. If they don't it's most likely because
+      # of a hang, in which case we'd like to know more quickly than the default 6hr timeout
+      # allows.
+      timeout-minutes: 60
+      run: |
+        echo "::add-matcher::./.github/workflows/main/problemMatchers/unittest.json"
+        ${{ matrix.testRunner }} "${{ env.GAFFER_BUILD_DIR }}/bin/gaffer test GafferTest.FileSystemPathTest"
+        echo "::remove-matcher owner=unittest::"
+      if: runner.os == 'Windows'
```

Comment:
I like the way you did this for Cortex, where `${{ matrix.tests }}` allows customisation without duplicating any build steps. If we called it `${{ matrix.testArguments }}` I could see it also being handy in future for passing a `-dontRunTheReallySlowPerformanceTests` argument for the debug builds.

Reply:
Fixed in e7478802556ee5db3f6c008b0ac07c06a33a563b